via/templates/pdf_wrapper.html.jinja2

```diff
@@ -35,15 +35,15 @@ See https://github.com/adobe-type-tools/cmap-resources
     <title>PDF.js viewer</title>{% block head %}{% endblock %}
 
 
-    <link rel="stylesheet" href="viewer.css">
+    <!-- <link rel="stylesheet" href="viewer.css"> -->
 
 
 <!-- This snippet is used in production (included from viewer.html) -->
 <link rel="resource" type="application/l10n" href="locale/locale.properties">
-<script src="../build/pdf.js"></script>
+<!-- <script src="../build/pdf.js"></script> -->
 
 
-    <script src="viewer.js"></script>
+    <!-- <script src="viewer.js"></script> -->
```

Comment:
I guess this verifies that the `template.replace(item, f"<!-- {item} -->")` above works :)

Reply:
Basically, if we take them all out, then their placement and strategy is all contained within the inner PDF file, and not the wrapper, rather than being spread between two. If you want to change the behavior to experiment, it's much easier to scoot a line between the head and footer blocks, rather than having to:

1) Learn that there is a resource by reading the wrapper
2) Edit the commenting out script
3) Move the asset